_targets/build.project.armv7m7-imxrt106x

```diff
@@ -77,9 +78,9 @@ PREINIT_SCRIPT=(
 USER_SCRIPT=(
 	"kernel ${BOOT_DEVICE}"
 	"app ${BOOT_DEVICE} -x dummyfs xip1 ocram2"
-	"app ${BOOT_DEVICE} -x imxrt-multi xip1 ocram2"
+	"app ${BOOT_DEVICE} -x imxrt-multi xip1 ocram2;aips14;aips5"
 	"app ${BOOT_DEVICE} -x psh xip1 ocram2"
-	"app ${BOOT_DEVICE} imxrt-flash ocram2 ocram2"
+	"app ${BOOT_DEVICE} imxrt-flash ocram2 ocram2;flexspi"
```

Comment:
`imxrt-flash` uses `0x402a8000..+0x300` (FSPI1)  and `0x402a4000..+0x300` (FSPI2) to access registers and TX/RX FIFO which is `AIPS-3` range, access through `0x7f000000..0x7fffffff` is not used directly.

```suggestion
	"app ${BOOT_DEVICE} imxrt-flash ocram2 ocram2;aips14"
```
btw, I wonder how `imxrt-flash` started without access to its registers (see above), it should fail with memmgmt fault during access to it's registers.

Reply:
Let's test it. Maybe it won't access anything during init when we xip from the memory?

Reply:
imxrt-`flash` seems to do nothing with the registers on imxrt1064. `imxrt`-multi crashed successfully when I took it's access the i/o away:
```
 Exception: 4 #MemMgtFault
   r0=00000558  r1=0000007f  r2=40184000  r3=2024e1bc
   r4=00000000  r5=2024dcf0  r6=70036ebc  r7=2024dcf0
   r8=2024dcf0  r9=2024cfa4 r10=00000000 r11=70036f4c
  r12=2024e1a2 psr=01000000  lr=7002e6ed  pc=7002db5c
  psp=2024fcd0 msp=2000b800 exr=ffffffed bfa=4018402c
  cfs=00000082 mma=4018402c
```

Reply:
I changed `flexspi` to `aips14` as advised

Reply:
`app ${BOOT_DEVICE} imxrt-flash ..` is without `-x`, that's why

Reply:
Haha, that explains it. I'll enable it

Reply:
Well, that's why it was disabled I guess:
```
 phoenix-rtos-devices/storage/imxrt-flash/flashsrv.c:871 error: imxrt-flashsrv: create_dev /dev/flash0 - err: -17
  phoenix-rtos-devices/storage/imxrt-flash/flashsrv.c:997 error: imxrt-flashsrv: flash memories were not initialized correctly.
```

Reply:
Not is scope of this PR

Reply:
Yep, right. Looks like missing partition in the last sector.